srcpkgs/KataGo-OpenCL/template

```diff
@@ -0,0 +1,41 @@
+# Template file for 'KataGo-OpenCL'
+pkgname=KataGo-OpenCL
+version=1.11.0
+revision=1
+#archs="i686 x86_64"
+wrksrc="KataGo-${version}"
+#create_wrksrc=yes
+build_wrksrc=cpp
+build_style=cmake
+configure_args="-DUSE_BACKEND=OPENCL -DNO_GIT_REVISION=1"
+
+if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
+	configure_args+=" -DCMAKE_CXX_STANDARD_LIBRARIES=-latomic"
+	makedepends+=" libatomic-devel"
+fi
+
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+#hostmakedepends=""
+makedepends="ocl-icd-devel zlib-devel libzip-devel"
+depends="mesa-opencl"
+short_desc="KataGo Go/Weiqi/Baduk analysis engine (OpenCL backend)"
+maintainer="Jason Manley <jason@jasondavid.tv>"
+license="MIT"
+homepage="https://github.com/lightvector/KataGo/"
+#changelog=""
+distfiles="https://github.com/lightvector/KataGo/archive/refs/tags/v${version}.tar.gz"
+checksum=3f63aa5dfaab861360fd6f9548aa7f552b007cac7e90c99089d3bb4bb4b9d451
+
+do_install() {
+	vbin build/katago
+}
+
+post_install() {
```

Comment:
`do_check` should go before `do_install`, same as execution order. No need to split post_install when do_install is defined in template.